### DISCUSSION TITLE
hach1238

### DISCUSSION THREAD

**Discussion by gerhardJaeger** [lingpy/are-sounds-sound-paper#2] [General]
In languages/hattorijaponic.tsv is a doculect 'hach1238' which does not occur in the current version of Glottolog. What shall we do with it? For now, I just left it out when constructing the reference tree.

**Comment by LinguList** [comment]
It is https://glottolog.org/resource/languoid/id/hach1239 now. We need to adjust our data accordingly, then re-compile. Leaving it out for now is also fine.

**Comment by LinguList** [comment]
The issue is fixed now in original data. But we need to re-run the code.

**Comment by LinguList** [comment]
Code now also re-run. If it works now without tweaking, we can close the issue.

**Reply by gerhardJaeger** [reply]
Done.
